src/NUnitFramework/tests/Constraints/CollectionEquivalentConstraintTests.cs

```diff
@@ -157,7 +157,7 @@ public void EquivalentHonorsUsing()
             ICollection set2 = new SimpleObjectCollection("z", "Y", "X");
 
             Assert.That(new CollectionEquivalentConstraint(set1)
-                .Using<string>((x, y) => StringUtil.Compare(x, y, true))
+                .Using<string>((x, y) => StringComparer.InvariantCultureIgnoreCase.Compare(x, y))
```

Comment:
This is just testing the Using, and doesn't need to be doing anything culture-sensitive anyway.